internal/action/variable.go

```diff
@@ -10,7 +16,53 @@ type VariablesInputItem struct {
 	Description string `yaml:"description,omitempty"`
 	Category    string `yaml:"category,omitempty"`
 	Sensitive   bool   `yaml:"sensitive,omitempty"`
+}
+
+type Variables []VariablesItem
+
+type VariablesItem struct {
+	Key         string
+	Value       string
+	Description string
+	Category    string
+	Sensitive   bool
+	Workspace   *Workspace
+}
+
+func NewVariablesItem(vi VariablesInputItem, w *Workspace) *VariablesItem {
+	return &VariablesItem{
+		Key:         vi.Key,
+		Value:       vi.Value,
+		Description: vi.Description,
+		Category:    vi.Category,
+		Sensitive:   vi.Sensitive,
+		Workspace:   w,
+	}
+}
+
+func (avi VariablesItem) ToResource() *tfeprovider.Variable {
```

Comment:
Stick to the same receiver name

Reply:
ah whoops, left over after a naming refactor (my first idea was ActionVariables which is not a good name)